tests/test_models.py

```diff
@@ -751,10 +750,10 @@ def tearDown(self):
 
     def test_model_obj_creation_with_course_and_org(self):
         """
-        Verify that an `IntegrityError` is raised if user try to create a
+        Verify that an `ValidationError` is raised if user try to create a
         SkillValidationConfiguration with both course and organization.
         """
-        with pytest.raises(IntegrityError) as raised_exception:
+        with pytest.raises(ValidationError) as raised_exception:
```

Comment:
Django 4.2 raises `ValidationError` before reaching to the point where `IntegrityError` is raised since the model's `full_clean()` method is catching this issue before database-level integrity checks can be enforced.

Reply:
I think this needs some manual verification? I will find some time to test this on my end.